tests/compiler_env_test.py

```diff
@@ -34,6 +34,30 @@ def test_benchmark_set_in_reset(env: LlvmEnv):
     assert env.benchmark == "benchmark://cbench-v1/dijkstra"
 
 
+def test_reward_space_setter(env: LlvmEnv):
+    env.reward_space = "IrInstructionCount"
+    assert env.reward_space != "IrInstructionCount"
+    env.reset()
+    assert env.reward_space == "IrInstructionCount"
```

Comment:
For this and the other 4 assertions you added, you must convert `env.reward_space` and `env.observation_space` to strings to check their value:

```suggestion
    assert str(env.reward_space) == "IrInstructionCount"
```

Reply:
Oh my bad, fixed them.